framework/src/main/kotlin/dev/alpas/ozone/PostgreSQLConnection.kt

```diff
@@ -0,0 +1,32 @@
+package dev.alpas.ozone
+
+import com.zaxxer.hikari.HikariDataSource
+import dev.alpas.Environment
+import me.liuwj.ktorm.database.Database
+import me.liuwj.ktorm.support.postgresql.PostgreSqlDialect
+
+@Suppress("unused")
+open class PostgreSQLConnection(env: Environment, config: ConnectionConfig? = null) : DatabaseConnection {
```

Comment:
@ashokgelal for consistency, would it be worth using `PostgreSqlConnection` similar to `MySqlConnection` instead of `SQL`?

Reply:
I agree. Thank you!